src/Pages/ChangePassword/ChangePassword.tsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react'
 import './ChangePassword.module.scss'
 
 const ChangePassword = () => {
```

Comment:
This component need to be deleted, as we are now using a modal for the ChangePassword.